rpi_csdt_community/settings.py

```diff
@@ -368,10 +362,15 @@
 
     # Make sure a GOOGLE_API_KEY is defined
     try:
-        GOOGLE_API_KEY  # noqa: F405
+        GOOGLE_API_KEY = None  # noqa: F405
```

Comment:
I think you're still missing what we're trying to tell you.
These statements are for error catching. Not for definition. You need to move these definition statements into the except of local_settings import (so that if there is not a local_settings it will use none).

You also need to put local_settings import above these for them to have any value.